cmd/machine-config-controller/start.go

```diff
@@ -82,6 +89,13 @@ func runStartCmd(_ *cobra.Command, _ []string) {
 			ctrlctx.FeatureGateAccess,
 		)
 
+		// create webhook server
+		webHookServer := webhook.NewServer(
+			webhook.NewConfig(defaultWebhookPort, defaultWebhookCertdir),
+			ctrlctx.InformerFactory.Machineconfiguration().V1alpha1().PinnedImageSets(),
```

Comment:
will this be featuregated? I know it isn't a crime to access a v1alpha1 api outside of a FG, but its good practice.

Reply:
Its featuregated in the server on Run it will return but I can do it earlier.